docs/audio/Samplebank_XML.md

```diff
@@ -0,0 +1,77 @@
+# Samplebank XML Format Specification
+
+Samplebank XMLs describe a samplebank file that contains compressed waveform data. It specifies which sample files to include as well as certain global properties such as the index of this samplebank.
+
+---
+
+```xml
+<SampleBank
+    Name="<C Indentifier>"
+    Index="<uint>"
+    Medium="<Medium>"
+    CachePolicy="<CachePolicy>"
+    BufferBug="[bool]"
+>
+```
+Begins a new samplebank.
+
+**Properties**
+
+- **Name**: The name of the samplebank.
+- **Index**: The index of the samplebank for the samplebank table. Must be a unique index for all samplebanks and pointers.
+- **Medium**: The storage medium, from the SampleMedium enum.
+- **CachePolicy**: The cache policy, from the AudioCacheLoadType enum.
```

Comment:
super nit: in the soundfont XML you put `SampleMedium` and `AudioCacheLoadType` in monospace font